packages/react/src/runtime/shorthand.ts

```diff
@@ -0,0 +1,88 @@
+import type { Depths, ShorthandProperties } from '@compiled/utils';
+
+// Copied from packages/utils/src/shorthand.ts so that we avoid
+// inflating the bundle size of @compiled/react/runtime with the contents
+// of @compiled/utils
+//
+// Keep this `shorthandBuckets` in sync with the `shorthandBuckets` defined in
+// packages/utils/src/shorthand.ts
+const shorthandBuckets: Record<ShorthandProperties, Depths> = {
+  all: 0,
+  animation: 1,
+  'animation-range': 1,
+  background: 1,
+  border: 1,
+  'border-block': 2,
+  'border-block-end': 3,
+  'border-block-start': 4,
+  'border-bottom': 5,
+  'border-color': 6,
+  'border-image': 1,
+  'border-inline': 7,
+  'border-inline-end': 8,
+  'border-inline-start': 9,
+  'border-left': 10,
+  'border-radius': 1,
+  'border-right': 10,
+  'border-style': 11,
+  'border-top': 12,
+  'border-width': 13,
```

Comment:
I'm not quite seeing this, I think this would be more like this:
```suggestion
  border: 1,
  'border-block': 3,
  'border-block-end': 5,
  'border-block-start': 5,
  'border-bottom': 4,
  'border-color': 2,
  'border-image': 2,
  'border-inline': 3,
  'border-inline-end': 5,
  'border-inline-start': 5,
  'border-left': 4,
  'border-radius': 1,
  'border-right': 4,
  'border-style': 2,
  'border-top': 4,
  'border-width': 2,
```

 - `border` > everything else (`border-radius` is different)
 - `border-color|width|style` > everything else
 - `border-inline|block` > `border-left|right|top|bottom`
 - `border-top` > `border-inline-start` — technically could leave this out if you want and drop this layer (make them even, but not sure how we'd fix them later)
 - `border-top-color` or `border-inline-start-color` always loses ⚠️ _maybe_ we should sort these consistently against each other, but I don't know…"logical property sorting" is kind of different, let's leave it out — could add and move things into a 6th layer later on, if we care

Reply:
ah good idea 👍 

manually configuring the bucket numbers > writing custom scripts to calculate the bucket numbers after all 🏆